src/base/QXmppJingleIq.cpp

```diff
@@ -990,6 +1028,13 @@ void QXmppJingleIq::Reason::parse(const QDomElement &element)
             break;
         }
     }
+
+    for (int i = NoErrorCondition + 1; i < JINGLE_RTP_ERROR_CONDITIONS.size(); i++) {
+        if (!element.firstChildElement(JINGLE_RTP_ERROR_CONDITIONS.at(i)).isNull()) {
+            m_rtpErrorCondition = RtpErrorCondition(i);
+            break;
+        }
+    }
```

Comment:
I think a better approach is to
1. find the first element with error namespace
2. using JINGLE_RTP_ERROR_CONDITIONS.indexOf() with the tag name
3. using NoErrorCondition as fallback (when no element could be found or indexOf() returned -1)